plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/gru_cell.cpp

```diff
@@ -11,21 +11,26 @@ using namespace LayerTestsDefinitions;
 
 namespace {
 std::vector<bool> should_decompose{
-    // false,
+    // true will decompose GRUCell to component ops and skip plaidml GRUCell implementation
     true,
+    false,
 };
 std::vector<size_t> batch{5};
 std::vector<size_t> hidden_size{
     1,
     10,
 };
 std::vector<size_t> input_size{
-    1,
+    3,
     30,
 };
+// When set `relu` as Gate gate activation and forbid clip, it
+// potentially will get a big error.
```

Comment:
From the last stage formula of gru cell, `Ht = (1 - zt) (.) ht + zt (.) Ht-1`, if there are no clipping to limit value upper bound, values of `ht` will reach about `n * 10^2` which will enlarge the error of output more than 100 times. Intuitively, this will cause this error.

I will check in details about this issue and try to avoid this situation.

Reply:
I see. This makes sense (especially in the FP16 case, which I presume is what is failing). I presume the test has an absolute (rather than relative) error threshold.

Given this cause, I think it would be acceptable to land this PR with the clip-free case commented out (i.e., as the PR currently is). Of course if there is a way to improve the precision that is preferable.

Reply:
Yes, this issue is in the FP16 case, I think the difference may comes from the fact that OpenVino nGraph interpreter actually does not support `fp16` but using `fp32` to do calculation instead.

From the `Compare` function in [`layer_test_utils`](https://github.com/plaidml/openvino/blob/fac0e2f0ee48e5cbf2656e8c304a82b592876f98/inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp#L99), it compares both absolute error and relative error, but share the [same value 0.01](https://github.com/plaidml/openvino/blob/fac0e2f0ee48e5cbf2656e8c304a82b592876f98/inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp#L25).

FYI, below is outputs of the case with error:
```
expected output:
data type=FP32
4.76217 1.19275 8 5 5

actual output: 
data type=FP32
4.76953 1.20898 8 5 5

Relative comparison of values expected: 1.19275 and actual: 1.20898 at index 1 with threshold 0.01 failed
index 1 absoluteDifference: 0.0162368
index 1 relativeDifference: 0.0134301

```

Reply:
Sounds good. Perhaps add to the comment in-code that the precision error seems potentially related to differences between fp16 PlaidML and fp32 reference implementations.

Reply:
Added